loopdb/loopout.go

```diff
@@ -155,6 +160,17 @@ func deserializeLoopOutContract(value []byte, chainParams *chaincfg.Params) (
 		contract.UnchargeChannel = &unchargeChannel
 	}
 
+	err = binary.Read(r, byteOrder, &contract.MaxSwapWaitTimeSec)
+	switch {
+	// Field not available for old versions.
+	case err == io.ErrUnexpectedEOF:
+		fallthrough
+	case err == io.EOF:
+		return &contract, nil
+	case err != nil:
+		return nil, err
+	}
```

Comment:
There is a migration mechanism already in place to which this migration can be added. It is slightly more work, but prevents this legacy code from staying around forever.